django/core/validators.py

```diff
@@ -312,6 +312,8 @@ def validate_ipv4_address(value):
 
 
 def validate_ipv6_address(value):
+    if isinstance(value, ipaddress.IPv6Address):
```

Comment:
Nice, from the ticket I thought you were going to change the helpers in `utils/ipv6.py`. This is nicer, thank you!